src/main/java/com/internet/shop/controllers/InjectDataController.java

```diff
@@ -25,9 +27,11 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         User user1 = userService.create(new User("User1", "log1", "111"));
-        User user2 = userService.create(new User("User2", "log2", "222"));
-        ShoppingCart shoppingCart1 = shoppingCartService.create(new ShoppingCart(user1.getId()));
-        ShoppingCart shoppingCart2 = shoppingCartService.create(new ShoppingCart(user2.getId()));
+        user1.setRoles(Set.of(Role.of("USER")));
+        User admin = userService.create(new User("admin", "admin", "222"));
+        admin.setRoles(Set.of(Role.of("ADMIN")));
+        shoppingCartService.create(new ShoppingCart(user1.getId()));
+        shoppingCartService.create(new ShoppingCart(admin.getId()));
```

Comment:
admin should not have his own shopping cart

also, let`s first set roles, and then save users to storage